apps/contract-sync-job/src/main.ts

```diff
@@ -1 +1,183 @@
-console.log('Hello World!');
+import { GovrnContract, NetworkConfig } from '@govrn/govrn-contract-client';
+import { GovrnGraphClient } from '@govrn/govrn-subgraph-client';
+import { GovrnProtocol, SortOrder } from '@govrn/protocol-client';
+import { ethers } from 'ethers';
+import { GraphQLClient } from 'graphql-request';
+
+const protcolUrl = process.env.PROTOCOL_URL;
+const ENDPOINT = process.env.SUBGRAPH_URL;
+const jobName = 'contract-sync-job';
+
+const contractAddress = '0xefad47a6888762b666727cae19d5e502f0a68f43';
+const networkConfig: NetworkConfig = {
+  address: contractAddress,
+  chainId: 2,
+};
+
+const provider = new ethers.providers.JsonRpcProvider(
+  'https://rinkeby.infura.io/v3/cda40b6016f04aa48da096c096068718'
```

Comment:
It would probably be good to make both this and the contract address config variables